content/tokio/tutorial/streams.md

```diff
@@ -394,16 +394,16 @@ stream! {
 ```
 
 [iter]: https://doc.rust-lang.org/book/ch13-02-iterators.html
-[`Stream`]: https://docs.rs/tokio-stream/0.1/tokio/trait.Stream.html
+[`Stream`]: https://docs.rs/tokio-stream/0.1/tokio_stream/trait.Stream.html
```

Comment:
```suggestion
[`Stream`]: https://docs.rs/futures-core/0.3/futures_core/stream/trait.Stream.html
```

https://docs.rs/tokio-stream/0.1/tokio_stream/trait.Stream.html is still broken

Reply:
Thank you for reviewing.
I did not notice it.

I applied your suggestion.